opt_einsum/parser.py

```diff
@@ -262,10 +262,16 @@ def convert_interleaved_input(operands: List[Any]) -> Tuple[str, List[Any]]:
     return subscripts, operands
 
 
-def parse_einsum_input(operands: Any) -> Tuple[str, str, List[ArrayType]]:
+def parse_einsum_input(operands: Any, **kwargs: Any) -> Tuple[str, str, List[ArrayType]]:
```

Comment:
Is this is in an internal function which does not have pass through kwargs, we can use `shapes: bool = False` rather than the kwarg unpacking to skip lines `297-302`.